tests/test_fft.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+import dpnp
+
+import numpy
+
+@pytest.mark.parametrize("type",
+                         [numpy.complex128, numpy.complex64, numpy.float32, numpy.float64, numpy.int32, numpy.int64],
+                         ids=['complex128', 'complex64', 'float32', 'float64', 'int32', 'int64'])
+def test_fft(type):
+    # 1 dim array
+    data = numpy.arange(100, dtype=type)

```

Comment:
Why you test foreign data type for this FFT test suite? it is not interoperability testing suite.